app/src/main/java/org/hisp/dhis/android/dashboard/ui/fragments/dashboard/DashboardViewPagerFragment.java

```diff
@@ -265,6 +267,13 @@ public void onResponseReceived(NetworkJob.NetworkJobResult<?> result) {
         }
     }
 
+    // Callback for No internet connection
+    NetworkUtils networkUtils = new NetworkUtils(this);
```

Comment:
Here you are implicitly capturing reference to Fragment through NetworkCallback implementation in class. It means, Activity to which Fragment is attached, will be leaked through static reference to callback in NetworkUtils.